### PR TITLE
Blog - Earthly v0.7(3 blog)

### PR DESCRIPTION
**Pull request by msali123**
Done with the required checks.

### CODE REVIEW

**Comment by msali123** [comment]
Preview URL: 
https://63e79e80c1305b33cf45a6f6--ephemeral-narwhal-3ac3b2.netlify.app

**Comment by msali123** [comment]
https://63e79e80c1305b33cf45a6f6--ephemeral-narwhal-3ac3b2.netlify.app/blog/launching-earthly-ci/
@jalletto please let me know if I should add supporting images to this blog

**Comment by msali123** [comment]
https://63e79e80c1305b33cf45a6f6--ephemeral-narwhal-3ac3b2.netlify.app/blog/launching-earthly-ci/
@jalletto please let me know if I should add supporting images to this blog

**Comment by msali123** [comment]
@jalletto any update on it?

**Comment by jalletto** [comment]
@msali123 I've given the team links to the three articles. Once they review I'll let you know.

File: blog/_posts/2023-02-21-launching-earthly-CI.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+categories:
+  - Tutorials
+toc: true
+author: Vlad
+
+internal-links:
+ - just an example
+---
+
+Hello world! We have partnered up with some [cool people in Silicon Valley LINK](LINK) [1] to fix the world of CI. So today we are launching [Earthly CI](https://earthly.dev/signup/earthly-ci), the world's first CI/CD solution that merges together a CI and a build system. A more fine-grained understanding of the build allows Earthly CI to run faster than a CI ever could before. And it's not an incremental improvement. It's a dramatic improvement. We're talking 100% to 2,000% faster. Here's how we did it.
```
[the remark this answers is not in the document]
Comment (jalletto) [reply blog/_posts/2023-02-21-launching-earthly-CI.md:12]: I don't think we ever covered how to do footnotes. We put footnotes at the very bottom, after the `{% include cta/cta1.html %}`. They are usually numbered and we can create links between them with [^1], [^2], [^3] and so on. 

This article has them if you want to see an example https://earthly.dev/blog/chroot/.

File: blog/_posts/2023-02-21-launching-earthly-CI.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+
+(Footnote)
+[1] Innovation Endeavors led our [$6.5M Seed+ round LINK](LINK), and were joined by 468 Capital and Uncorrelated Ventures. A number of founders of companies such as Cockroach Labs, DigitalOcean, Mesosphere, DataDog, Sentry, and Instana, plus a number of creators and maintainers of notable developer platforms, such as Docker, Elixir, VS Code, GitHub Copilot, Hashicorp, Envoy proxy, Cypress, Mesos and many others, have also previously invested in Earthly.
```
Comment (jalletto) [inline blog/_posts/2023-02-21-launching-earthly-CI.md:15]:
```suggestion

[^1] Innovation Endeavors led our [$6.5M Seed+ round LINK](LINK), and were joined by 468 Capital and Uncorrelated Ventures. A number of founders of companies such as Cockroach Labs, DigitalOcean, Mesosphere, DataDog, Sentry, and Instana, plus a number of creators and maintainers of notable developer platforms, such as Docker, Elixir, VS Code, GitHub Copilot, Hashicorp, Envoy proxy, Cypress, Mesos and many others, have also previously invested in Earthly.
```

Comment (jalletto) [reply blog/_posts/2023-02-21-launching-earthly-CI.md:15]: Move this section to below ` {% include cta/cta1.html %}`. Then confirm that the links work. The one in the paragraph above should take you down to the bottom where the footnote is. Then there should be a small in the form of an arrow at the end of the sentence that brings you back up to the top.

File: blog/_posts/2023-02-21-launching-earthly-CI.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+~~~
+
+Earthly allows you to execute CI/CD pipelines on your computer, thus allowing you to iterate fast when developing or maintaining them. Earthly CI was designed with the explicit goal of simplifying the dev-test cycle of CI/CD pipelines. (Our open-source users are particularly excited about this)
+
+In Earthly CI, every step involved in the build is cached automatically, and the cache is available instantly. No upload or download necessary. The cache always makes the build faster, there's no guesswork needed, and for that reason the cache is always on for every step of every pipeline.
+
+The syntax also allows the declaration of interdependencies. The interdependencies allow the system to build a directed acyclic graph (DAG) on the fly, and execute it with high parallelism. [2]
+
+(Footnote)
+[2] And because everything runs in containers, it is possible in the future to also distribute a single build across a compute cluster for maximum parallelization. Today, Earthly CI only runs on one machine at a time, however.
```
Comment (jalletto) [inline blog/_posts/2023-02-21-launching-earthly-CI.md:127]: Same thing with this footnote as above. Move the actual footnote to the bottom and confirm that the links work properly.

File: blog/_posts/2023-02-21-earthly-v0-7.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+## Compatibility With Earthly CI
+
+Earthly 0.7 is the first version compatible with [Earthly CI](https://earthly.dev/product/earthly-ci). All Earthly CI pipelines are defined in Earthfiles using Earthly's simple, familiar syntax and stored in the project's repository.
+
+Earthly 0.7 introduces the new keywords `PIPELINE` and `TRIGGER` to define Earthly CI pipelines. You can execute pipelines on your computer in the exact same way you execute build targets. So it's super simple to iterate on and test CI pipelines on your computer before you push them to prod.
+
+The following example defines a CI pipeline named `build-pipeline` that executes the `+my-service` target and is triggered whenever there is a pull request raised against the `main` branch of the repository:
+
+~~~{.earthfile caption=""}
```
Comment (jalletto) [inline blog/_posts/2023-02-21-earthly-v0-7.md:24]:
```suggestion
~~~{.Dockerfile caption=""}
```

Comment (jalletto) [reply blog/_posts/2023-02-21-earthly-v0-7.md:24]: Can you change all the .earthfile to .Dockerfile?

**Review by jalletto — CHANGES_REQUESTED** [review]

File: blog/_posts/2023-02-21-launching-earthly-CI.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+## Problem #2: Misaligned Incentives
+
+> The slower your build, the more the vendor profits
+
+As an industry, we have gotten used to consuming CI/CD by-the-minute. This worked fine for the previous era when CI/CD was merely a glorified bash script runner.
+
+Nowadays, however, paying by compute time means that the slower your build, the more the vendor profits. This isn't right, because it creates a perverse incentive for the CI/CD vendor to never innovate in providing ways to speed things up. If a CI/CD tool is meant to deliver productivity gains, then correlating the value to how much less productive you are is the wrong way to go.
+
+## Problem #3: Ci/Cd Pipelines Are a Nightmare to Maintain
```
Comment (vladaionescu) [inline blog/_posts/2023-02-21-launching-earthly-CI.md:45]:
```suggestion
## Problem #3: CI/CD Pipelines Are a Nightmare to Maintain
```

File: blog/_posts/2023-02-21-launching-earthly-CI.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+
+And finally, Earthly CI was designed to work with any code layout. And more specifically, it was designed and tested with both monorepos and polyrepos in mind. For monorepos, Earthly understands the interdependence of build targets, and the specific source files that contribute to those targets, allowing it cache and parallelize builds accordingly. If you're iterating on an integration test and you only change 1 out of 7 microservices, Earthly only rebuilds that 1 microservice before re-running the integration test. Similarly, Earthly has a strong understanding of setups where the build is split across many repositories. It is Git-hash-aware, and will only rebuild what has actually changed.
```
Comment (vladaionescu) [inline blog/_posts/2023-02-21-launching-earthly-CI.md:135]:
```suggestion
And finally, Earthly CI was designed to work with any code layout. And more specifically, it was designed and tested with both monorepos and polyrepos in mind. For monorepos, Earthly understands the interdependence of build targets, and the specific source files that contribute to those targets, allowing it to cache and parallelize builds accordingly. If you're iterating on an integration test and you only change 1 out of 7 microservices, Earthly only rebuilds that 1 microservice before re-running the integration test. Similarly, Earthly has a strong understanding of setups where the build is split across many repositories. It is Git-hash-aware, and will only rebuild what has actually changed.
```

**Comment by msali123** [comment]
Preview URL:
https://63ef4dfba54be84fe721aed9--ephemeral-narwhal-3ac3b2.netlify.app

**Comment by adamgordonbell** [comment]
@msali123 This PR will probably have a lot of tweaks requested over the next few days because we want to get it right. 

Can you change the publish dates of the articles (rename the files) as follows:
2/23/2023 -> Launching Earthly CI
2/23/2023 ->Announcing Earthly's $6.5M Seed+ round
2/28/2023 -> Earthly 0.7

File: blog/_posts/2023-02-21-earthly-v0-7.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+|--explicit-global |    Base target args must have a --global flag in order to be considered global args|
+|--check-duplicate-images    | Check for duplicate images during output|
+|--earthly-version-arg    | Enables builtin ARGs: EARTHLY_VERSION and EARTHLY_BUILD_SHA|
+|--use-cache-command    | Allow use of CACHE command in Earthfiles|
+|--use-host-command    | Allow use of HOST command in Earthfiles|
+|--use-copy-link    | Use the equivalent of COPY --link for all copy-like operations|
+|--new-platform    | Enable new platform behavior|
+|--no-tar-build-output    | Do not print output when creating a tarball to load into WITH DOCKER|
+|--use-no-manifest-list    | Enable the SAVE IMAGE --no-manifest-list option|
+|--use-chmod    | Enable the SAVE IMAGE --no-manifest-list option|
```
Comment (adamgordonbell) [inline blog/_posts/2023-02-21-earthly-v0-7.md:128]:
```suggestion
|--use-chmod    | Allow setting the permissions of the copied files.|
```

File: blog/_posts/2023-02-21-earthly-v0-7.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+categories:
+  - Articles
+toc: true
+author: Vlad
+
+internal-links:
+ - just an example
+---
+
+Today, we're announcing the release of Earthly v0.7. The new version introduces changes that make Earthly work with Earthly CI, our new CI/CD platform (read more about Earthly CI in [Earthly CI](TODO: Link to CI launch blog post)), in addition to promoting several features from Experimental or Beta to GA – notably Podman support and the `WAIT` command.
```
Comment (adamgordonbell) [inline blog/_posts/2023-02-21-earthly-v0-7.md:12]: 'TODO: Link to CI launch blog post' should be the other post in this PR

File: blog/_posts/2023-02-21-earthly-v0-7.md [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+
+We do not take major or minor releases lightly at Earthly. We've never had a major release, and our last minor release was over a year ago, in December 2021 (read about it in [Announcing Earthly v0.6](https://earthly.dev/blog/earthly-v0-6/)). That's because we know that the reliability and stability of your build and CI/CD processes are of the utmost importance. In every minor release of Earthly (and also eventually when we have a major release), all features promoted to GA have finalized APIs and have been through thorough testing. That's why we are comfortable enabling all features promoted to GA by default.
+
+## Compatibility With Earthly CI
+
+Earthly 0.7 is the first version compatible with [Earthly CI](https://earthly.dev/product/earthly-ci). All Earthly CI pipelines are defined in Earthfiles using Earthly's simple, familiar syntax and stored in the project's repository.
```
Comment (adamgordonbell) [inline blog/_posts/2023-02-21-earthly-v0-7.md:18]: @thtmnisamnstr  I feel like sending them to the CI blog post here would be better? What do you think?

File: blog/_posts/2023-02-21-new-fundings-at-earthly.md [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+
+Today, we are announcing Earthly's $6.5M Seed+ funding, taking our total capital raised to date to $8.6M. The round was led by Davis Treybig from Innovation Endeavors, with participation from Florian Leibert of 468 Capital, and Salil Deshpande of Uncorrelated Ventures.
+
+Earthly's complete set of investors includes founders of companies such as Cockroach Labs, DigitalOcean, Mesosphere, DataDog, Sentry, and Instana, as well as creators and maintainers of notable developer platforms, such as Docker, Elixir, VS Code, GitHub Copilot, Hashicorp, Envoy proxy, Cypress, Mesos, and others.
+
+The new funding will be used to improve and add more enterprise-friendly features to [Earthly CI LINK](Link to other blog post). Earthly CI is a fast, repeatable CI/CD platform that is super simple to use. It gives teams repeatable CI pipelines that you write once and run anywhere; has an automatic and instantly available build cache that makes builds faster; and it's incredibly user-friendly.
```
Comment (adamgordonbell) [inline blog/_posts/2023-02-21-new-fundings-at-earthly.md:18]: This should link to the CI article, in same PR

**Review by adamgordonbell — CHANGES_REQUESTED** [review]

**Comment by jalletto** [comment]
@msali123 For the banner images, they look low res. Are they the same dimensions we normally use? Can you try to use a higher resolution image? 

For the launching-earthly-ci, the earthly logo on the rocket isn't cropped so it looks tacked on. Can we just replace the rocket with a high resolution version of the logo? Here is a link to a png https://drive.google.com/file/d/1JHYwKTAwKoQyghgIz1Tdgq6mITLmgyO4/view?usp=sharing. 

And then for the Announcing Earthly’s $6.5M Seed+ round, can you put the logo in a white circle and then leave some room around the logo and the edge of the circle so it doesn't look like the logo is spilling out? Similar to how the f in this facebook logo sits in the circle. https://www.freeiconspng.com/img/46264.

**Comment by msali123** [comment]
> @msali123 This PR will probably have a lot of tweaks requested over the next few days because we want to get it right. 
> 
> Can you change the publish dates of the articles (rename the files) as follows:
> 2/23/2023 -> Launching Earthly CI
> 2/23/2023 ->Announcing Earthly's $6.5M Seed+ round
> 2/28/2023 -> Earthly 0.7
> 

Sure @adamgordonbell I'll change them to make this perfect.

File: blog/_posts/2023-02-21-earthly-v0-7.md (hunk 8 above)
Comment (msali123) [reply blog/_posts/2023-02-21-earthly-v0-7.md:12]: Link to - Launching Earthly CI
@adamgordonbell ?

Comment (msali123) [reply blog/_posts/2023-02-21-earthly-v0-7.md:12]: ??

File: blog/_posts/2023-02-21-earthly-v0-7.md (hunk 9 above)
Comment (msali123) [reply blog/_posts/2023-02-21-earthly-v0-7.md:18]: ?? @jalleto

**Comment by msali123** [comment]
@jalletto  
Preview URL: https://63f510917bdf8011e1a80508--ephemeral-narwhal-3ac3b2.netlify.app

File: blog/_posts/2023-02-21-earthly-v0-7.md (hunk 9 above)
Comment (jalletto) [reply blog/_posts/2023-02-21-earthly-v0-7.md]: @thtmnisamnstr bump

File: blog/_posts/2023-02-21-earthly-v0-7.md (hunk 8 above)
Comment (jalletto) [reply blog/_posts/2023-02-21-earthly-v0-7.md]: yes nice job Sameer 👍

File: blog/_posts/2023-02-21-earthly-v0-7.md (hunk 9 above)
Comment (thtmnisamnstr) [reply blog/_posts/2023-02-21-earthly-v0-7.md]: I agree. Let's change the link to the Earthly CI launch blog post.

File: blog/_posts/2023-02-23-launching-earthly-CI.md [hunk 11]
```diff
@@ -0,0 +1,143 @@
...
+        RUN go test --tags=integration ./...
+    END
+
+src:
+    COPY go.mod go.sum ./
+    RUN go mod download
+    COPY main.go ./
+    RUN go build -o my-service main.go
+    SAVE ARTIFACT my-service AS LOCAL my-service
+~~~
```
Comment (vladaionescu) [inline blog/_posts/2023-02-23-launching-earthly-CI.md:115]: Is there a way to use Earthly-specific syntax highlighting here? I know @jalletto mentioned that there might be a possibility to do this.

File: blog/_posts/2023-02-28-earthly-v0-7.md [hunk 12]
```diff
@@ -0,0 +1,148 @@
+---
+title: "Announcing Earthly v0.7"
+categories:
+  - Articles
+toc: true
+author: Vlad
```
Comment (vladaionescu) [inline blog/_posts/2023-02-28-earthly-v0-7.md:6]: I believe the author of this article is Gavin. CC @jalletto

File: blog/_posts/2023-02-23-launching-earthly-CI.md [hunk 13]
```diff
@@ -0,0 +1,143 @@
+---
+title: "Earthly CI: Launching a new era for CI"
+categories:
+  - Articles
```
Comment (vladaionescu) [inline blog/_posts/2023-02-23-launching-earthly-CI.md:4]: All three articles should have the category set as News.

Comment (msali123) [reply blog/_posts/2023-02-23-launching-earthly-CI.md:4]: done

File: blog/_posts/2023-02-23-launching-earthly-CI.md (hunk 11 above)
Comment (msali123) [reply blog/_posts/2023-02-23-launching-earthly-CI.md:115]: @jalletto could you help on this?

File: blog/_posts/2023-02-28-earthly-v0-7.md (hunk 12 above)
Comment (msali123) [reply blog/_posts/2023-02-28-earthly-v0-7.md:6]: There's no info about Gavin in Author @jalletto

**Comment by msali123** [comment]
Preview URL:
https://63f5d3891f7f8c59c0b971e1--ephemeral-narwhal-3ac3b2.netlify.app

File: blog/_posts/2023-02-23-launching-earthly-CI.md (hunk 11 above)
Comment (jalletto) [reply blog/_posts/2023-02-23-launching-earthly-CI.md:115]: @vladaionescu We don't have earthly specific highlighting. I Dockerfile is the best we can do.

Comment (vladaionescu) [reply blog/_posts/2023-02-23-launching-earthly-CI.md:115]: Ok sounds good

File: blog/_posts/2023-02-28-earthly-v0-7.md (hunk 12 above)
Comment (jalletto) [reply blog/_posts/2023-02-28-earthly-v0-7.md:6]: @msali123 I'm reaching out to get his bio. Ill let you know when I have it.

File: blog/_posts/2023-02-23-launching-earthly-CI.md (hunk 13 above)
Comment (jalletto) [reply blog/_posts/2023-02-23-launching-earthly-CI.md]: Thanks @msali123. That was my mistake telling you to use Article.

File: blog/_posts/2023-02-28-earthly-v0-7.md (hunk 12 above)
Comment (jalletto) [reply blog/_posts/2023-02-28-earthly-v0-7.md:6]: Actually, Gavin is going to make a PR against this one with the bio.

**Comment by jalletto** [comment]
@msali123 Couple of last things. (Thanks for your patience on this one!)

In [this file](https://github.com/earthly/website/blob/main/blog/_layouts/home.html#L4) can you change `hero_post: earthly-open-source` to be `hero_post: launching-earthly-ci` and then change `featured_post_a: docker-volumes` to be `featured_post_a: earthly-open-source`

**Comment by msali123** [comment]
@jalletto  no worries for the changes I'm happy to do them 😃 

I've commit this change but will push after adding the new author.

Is it okay?

**Comment by jalletto** [comment]
@msali123  I think Gavin is going to push the author profile himself, so I think you are good to go. I'm not going to merge this until tomorrow morning though since we don't want it to go out before then.

**Comment by msali123** [comment]
Preview URL: https://63f6817494fd94095be90969--ephemeral-narwhal-3ac3b2.netlify.app
@jalletto

File: blog/_layouts/home.html [hunk 14]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: home-archive
-hero_post: earthly-open-source
-featured_post_a: docker-volumes
+hero_post: launching-earthly-ci
```
Comment (jalletto) [inline blog/_layouts/home.html:3]:
```suggestion
hero_post: launching-earthly-CI
```

File: blog/_posts/2023-02-23-launching-earthly-CI.md [hunk 15]
```diff
@@ -0,0 +1,143 @@
...
+categories:
+  - News
+toc: true
+author: Vlad
+
+internal-links:
+ - just an example
+---
+
+Hello world! We have partnered up with some [cool people in Silicon Valley LINK](LINK) [^1] to fix the world of CI. So today we are launching [Earthly CI](https://earthly.dev/signup/earthly-ci), the world's first CI/CD solution that merges together a CI and a build system. A more fine-grained understanding of the build allows Earthly CI to run faster than a CI ever could before. And it's not an incremental improvement. It's a dramatic improvement. We're talking 100% to 2,000% faster. Here's how we did it.
```
Comment (jalletto) [inline blog/_posts/2023-02-23-launching-earthly-CI.md:12]:
```suggestion
Hello world! We have partnered up with some cool people in Silicon Valley [^1] to fix the world of CI. So today we are launching [Earthly CI](https://earthly.dev/signup/earthly-ci), the world's first CI/CD solution that merges together a CI and a build system. A more fine-grained understanding of the build allows Earthly CI to run faster than a CI ever could before. And it's not an incremental improvement. It's a dramatic improvement. We're talking 100% to 2,000% faster. Here's how we did it.
```

**Review by jalletto — APPROVED** [review]

File: blog/_posts/2023-02-28-earthly-v0-7.md [hunk 16]
```diff
@@ -0,0 +1,149 @@
...
+
+For more information on using Podman with Earthly, see [our docs](https://docs.earthly.dev/docs/guides/podman).
+
+## `WAIT` Command
+
+***Breaking Change**: The behavior of `--push` mode has changed in a backwards incompatible manner.*
+
+Pushing no longer requires everything else to succeed. Previously, `--push` commands would only execute if all other commands had succeeded. This requirement is no longer enforced. This allows for more flexible push ordering using the new `WAIT` command. To achieve the behavior of the previous `--push` mode, you need to wrap any pre-required commands in a `WAIT`/`END` block.
+
+### Example: Push an image only if tests have passed
```
Comment (jalletto) [inline blog/_posts/2023-02-28-earthly-v0-7.md:79]:
```suggestion
### Example: Push an Image Only if Tests Have Passed
```

File: blog/_posts/2023-02-23-launching-earthly-CI.md [hunk 17]
```diff
@@ -0,0 +1,143 @@
...
+categories:
+  - News
+toc: true
+author: Vlad
+
+internal-links:
+ - just an example
+---
+
+*Hello world! We have partnered up with some [cool people in Silicon Valley](/blog/new-fundings-at-earthly/) to fix the world of CI. So today we are launching [Earthly CI](https://earthly.dev/signup/earthly-ci), the world's first CI/CD solution that merges together a CI and a build system. A more fine-grained understanding of the build allows Earthly CI to run faster than a CI ever could before. And it's not an incremental improvement. It's a dramatic improvement. We're talking 100% to 2,000% faster. Here's how we did it.*
```
Comment (jalletto) [inline blog/_posts/2023-02-23-launching-earthly-CI.md:12]:
```suggestion
*Hello world! We have partnered up with some [cool people in Silicon Valley](/blog/new-fundings-at-earthly/) [^1] to fix the world of CI. So today we are launching [Earthly CI](https://earthly.dev/signup/earthly-ci), the world's first CI/CD solution that merges together a CI and a build system. A more fine-grained understanding of the build allows Earthly CI to run faster than a CI ever could before. And it's not an incremental improvement. It's a dramatic improvement. We're talking 100% to 2,000% faster. Here's how we did it.*
```

File: blog/_posts/2023-02-23-launching-earthly-CI.md [hunk 18]
```diff
@@ -0,0 +1,143 @@
...
+## Get Started With Earthly Ci
+
+Earthly CI is being made available today. As the platform is new, we are letting new users in gradually, to ensure overall platform stability.
+
+If this is interesting to you, [get started with Earthly CI](https://earthly.dev/signup/earthly-ci)! Let's build the next generation of software together!
+
+{% include cta/cta1.html %}
+
+[^1]:
+    Innovation Endeavors led our [$6.5M Seed+ round LINK](LINK), and were joined by 468 Capital and Uncorrelated Ventures. A number of founders of companies such as Cockroach Labs, DigitalOcean, Mesosphere, DataDog, Sentry, and Instana, plus a number of creators and maintainers of notable developer platforms, such as Docker, Elixir, VS Code, GitHub Copilot, Hashicorp, Envoy proxy, Cypress, Mesos and many others, have also previously invested in Earthly.
```
Comment (jalletto) [inline blog/_posts/2023-02-23-launching-earthly-CI.md:140]:
```suggestion
    Innovation Endeavors led our [$6.5M Seed+ round](/blog/new-fundings-at-earthly/), and were joined by 468 Capital and Uncorrelated Ventures. A number of founders of companies such as Cockroach Labs, DigitalOcean, Mesosphere, DataDog, Sentry, and Instana, plus a number of creators and maintainers of notable developer platforms, such as Docker, Elixir, VS Code, GitHub Copilot, Hashicorp, Envoy proxy, Cypress, Mesos and many others, have also previously invested in Earthly.
```

**Review by jalletto — APPROVED** [review]

**Pull request merged**